client/src/partials/fee_center/fee_center.html

```diff
@@ -0,0 +1,30 @@
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.FINANCE</li>
+      <li class="title" translate>TREE.FEE_CENTER</li>
+    </ol>
+
+    <div class="toolbar">
+      <div class="toolbar-item">
+        <button type="button"
+          class="btn btn-default"
+          data-method="create"
+          ui-sref="feeCenter.create">
+          <i class="fa fa-plus-circle"></i> {{ 'FORM.LABELS.ADD' | translate }}
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="flex-content">
+  <div class="container-fluid">
+    <div id="feeCenterListGrid"
+      class="grid-full-height"
+      ui-grid="FeeCenterCtrl.gridOptions"
```

Comment:
We have a fairly standard way of showing UI grid loading progress and error states - this helps the user know that either a large querry is in progress or something went wrong. It would be good to make use of that here.